x/auth/client/cli/sign.go

```diff
@@ -51,10 +52,10 @@ recommended to set such parameters manually.`,
 	cmd.Flags().Bool(flagValidateSigs, false, "Print the addresses that must sign the transaction, "+
 		"those who have already signed it, and make sure that signatures are in the correct order.")
 	cmd.Flags().Bool(flagOffline, false, "Offline mode. Do not query local cache.")
-	return cmd
+	return client.PostCommands(cmd)[0]
```

Comment:
ditto. Do we need this for a prefix or something? If so a helper function would make it clearer.

Reply:
Setting flags.

Reply:
Added!